tests/clickhouse-test

```diff
@@ -542,7 +543,8 @@ class SettingsRandomizer:
             0.2, 0.5, 1, 10 * 1024 * 1024 * 1024
         ),
         "local_filesystem_read_method": lambda: random.choice(
-            ["read", "pread", "mmap", "pread_threadpool", "io_uring"]
+            # io_uring is not supported on macOS, don't choose it
```

Comment:
uring is Linux-specific.

Let's change the logic "is not Darwin" to "is Linux".